enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotChat/MetabotChatEditor/MetabotChatEditor.unit.spec.tsx

```diff
@@ -0,0 +1,175 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { setupEnterprisePlugins } from "__support__/enterprise";
+import { setupDatabaseEndpoints } from "__support__/server-mocks";
+import { setupSearchEndpoints } from "__support__/server-mocks/search";
+import { mockSettings } from "__support__/settings";
+import { renderWithProviders } from "__support__/ui";
+import type { SuggestionModel } from "metabase-enterprise/rich_text_editing/tiptap/extensions/shared/types";
+import { createMockUser } from "metabase-types/api/mocks";
+import { createSampleDatabase } from "metabase-types/api/mocks/presets";
+import { createMockSearchResult } from "metabase-types/api/mocks/search";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { MetabotChatEditor } from "./MetabotChatEditor";
+
+const defaultProps = {
+  value: "",
+  onChange: jest.fn(),
+  onSubmit: jest.fn(),
+  suggestionModels: [
+    "table",
+    "database",
+    "card",
+    "dashboard",
+    "collection",
+  ] as SuggestionModel[],
+};
+
+const setup = (props = {}) => {
+  setupEnterprisePlugins();
+  const settings = mockSettings({ "site-url": "http://localhost:3000" });
+
+  return renderWithProviders(
+    <MetabotChatEditor {...defaultProps} {...props} />,
+    {
+      storeInitialState: createMockState({
+        settings,
+        currentUser: createMockUser(),
+      }),
+    },
+  );
+};
+
+const getEditor = () =>
+  // eslint-disable-next-line testing-library/no-node-access
+  document.querySelector('[contenteditable="true"]')! as HTMLElement;
```

Comment:
I'll accept it if this is the only option, but we're not able to grab this using roles or anything eh?